Model/Customer.php

```diff
@@ -0,0 +1,167 @@
+        $this->omise                   = $omise;
+        $this->customerAPI             = $customerAPI;
+
+        $this->omise->defineUserAgent();
+        $this->omise->defineApiVersion();
+        $this->omise->defineApiKeys();
+
+        $this->initializeObject();
+    }
+
+    /**
+     * Check whether the executor is logged in or not.
+     *
+     * @return bool
+     */
+    public function isLoggedIn()
+    {
+        return $this->getMagentoCustomerId() ? true : false;
+    }
+
+    /**
+     * Get an Omise customer id that belongs to
+     * a specific user who execute the method.
+     *
+     * @return string
+     */
+    public function getId()
+    {
+        return $this->magentoCustomer->getData(self::OMISE_CUSTOMER_ID_FIELD);
+    }
+
+    /**
+     * Get a Magento customer id that belongs to
+     * a specific user who execute the method.
+     *
+     * @return int|null
+     *
+     * @see    magento/module-customer/Model/Session.php
+     */
+    public function getMagentoCustomerId()
+    {
+        return $this->magentoCustomerSession->getCustomerId();
+    }
+
+    /**
+     * Create an Omise customer, then assigning its Omise customer id
+     * back to the customer's attribute who execute the method.
+     *
+     * @return self
+     */
+    public function create(array $params)
+    {
+        $this->customerAPI = $this->customerAPI->create($params);
+
+        $this->magentoCustomer->setData(self::OMISE_CUSTOMER_ID_FIELD, $this->customerAPI->id);
+        $this->magentoCustomerResource->saveAttribute($this->magentoCustomer, self::OMISE_CUSTOMER_ID_FIELD);
+
+        return $this;
+    }
+
+    /**
+     * Attach an Omise Token to a Customer object.
+     *
+     * @param string $cardToken
+     *
+     * @return self
+     */
+    public function addCard($cardToken)
+    {
+        if (! $this->getId()) {
+            $this->create(array(
+                'email'       => $this->magentoCustomer->getEmail(),
+                'description' => trim($this->magentoCustomer->getFirstname() . ' ' . $this->magentoCustomer->getLastname())
+            ));
+        }
+
+        $this->customerAPI = $this->customerAPI->update(array('card' => $cardToken));
+
+        return $this;
+    }
+
+    /**
+     * Retrieve all cards that belong to a specific Omise Customer object.
+     *
+     * @param  array $options
+     *
+     * @return \OmiseCardList
+     *
+     * @see    https://github.com/omise/omise-php/blob/master/lib/omise/OmiseCardList.php
+     */
+    public function cards($options = array())
+    {
+        return $this->customerAPI->cards($options);
+    }
+
+    /**
+     * @return array  of Omise card object.
+     *
+     * @see    https://www.omise.co/cards-api
+     */
+    public function getLatestCard()
+    {
+        $cards = $this->cards(array('order' => 'reverse_chronological'));
+
+        return $cards['total'] > 0 ? $cards['data'][0] : null;
```

Comment:
`> 0` unnecessary

Reply:
Note: As discussed, in a sense of number, `total > 0` does sounds more verbally and giving a bit more sense when it comes to readability (`if total is greater than 0 then ...` instead of `if total then ...`).

So we are going with this `> 0` approach.